src/cli/README_SRP_CLIENT.md

```diff
@@ -0,0 +1,300 @@
+# OpenThread CLI - SRP Client
+
+## Quick Start
+
+### Build with CoAP API support
```

Comment:
```suggestion
### Build with SRP Client support
```

Do you want to have this duplicated here as well as in README_SRP.md? Would a link be better?

Reply:
Good point. thanks. I actually wanted it to be only in `README_SRP.md`. Will remove it from here.